pkg/gatewayserver/io/basicstationlns/basicstationlns.go

```diff
@@ -75,13 +81,18 @@ func (s *srv) handleDiscover(c echo.Context) error {
 		return err
 	}
 
+	if req.EUI.IsZero() {
+		sendErrorMessageToStation(ws, logger, "Invalid request")
```

Comment:
Shouldn't we `defer` writing an error message, if any, when a message can't be handled? And distill the error message from the type?

Reply:
The error messages are sent back only for the discovery endpoint. The connection is simply closed for errors on the traffic endpoint.